### DISCUSSION TITLE
Errors when compiling with modules

### DISCUSSION THREAD

**Discussion by NeoStele** [azerothcore/azerothcore-wotlk#17629] [General]
Good evening,

I have been trying to get this to work for a little bit and haven't been very lucky.  I have the server up and running without any issues.  However, when I try to add any modules, I run into errors when building in Visual Studio.

The modules, downloaded from the cataloge, I'm trying to use are:
AH Bot
All races all classes
Learn Spells
PlayerBots
Progression System
Solo LFG
Transmog

I have also tried doing them separately, but run into similar issues with each one.  I'm kind of at a dead end and not really sure what else to do.  Please advise, any suggestions are welcome.

I have attached my full visual studio report, but will also copy and paste the last bit here:

> 22>Done building project "modules.vcxproj" -- FAILED.
25>------ Build started: Project: worldserver, Configuration: RelWithDebInfo x64 ------
25>Building Custom Rule F:/World of Warcraft/AC-Source/src/server/apps/CMakeLists.txt
25>cmake_pch.cxx
25>Main.cpp
25>ACSoap.cpp
25>CliRunnable.cpp
25>RASession.cpp
25>WheatyExceptionReport.cpp
25>ServiceWin32.cpp
25>LINK : fatal error LNK1181: cannot open input file '..\..\..\modules\RelWithDebInfo\modules.lib'
25>Done building project "worldserver.vcxproj" -- FAILED.
26>------ Build started: Project: ALL_BUILD, Configuration: RelWithDebInfo x64 ------
26>Building Custom Rule F:/World of Warcraft/AC-Source/CMakeLists.txt
========== Build: 24 succeeded, 2 failed, 0 up-to-date, 0 skipped ==========
========== Build started at 9:08 PM and took 16:42.144 minutes ==========
[ACVizioError.txt](https://github.com/azerothcore/azerothcore-wotlk/files/13233806/ACVizioError.txt)

The last little bit is what is throwing me off:
I can't seem to find a modules.lib in either my AC source folders, nor my Build folders.

**Comment by vnasing** [comment]
和我的问题一样，不知道应该怎么解决。干净的AC端可以成功编译，带有任何模块都会失败

**Comment by ryankennedy85** [comment]
Are you removing the -master from the folders before building?
